### ISSUE TITLE
2.5 metadata lists 2021 as final year

### ISSUE THREAD

**Issue by rgieseke** [JGuetschow/PRIMAP-hist#76]
The metadata of the 2.5 release (https://zenodo.org/records/10006301) lists 2021 as the final year in "Files included in the dataset":

> all time series to 2021

Should that be 2022?

It also says 

> The v2.5 release of PRIMAP-hist reduced the time-lag from 2 to 1 years.

The same sentence is already used in the 2.4 version.

Some other things I noted:

In "Sources" there seem to be some TeX/Markdown leftovers

> Global CO$_2$ emissions from cement production v220919 (Andrew 2022)**

Also `CO$_2$` in general.

Affiliations are listed as "Potsdam Institute for Climate Impact Reseach (PIK)" only, not sure if that should include Climate Resource?

**Comment by JGuetschow** [comment]
Thanks for spotting and letting me know. I think the problems are now all fixed on zenodo. I can't change the pdf, however, most of the issues are just from transferring changes from markdown to the zenodo format (zenodo doesn't handle html any more, so I had to transfer everything manually)

**Comment by rgieseke** [comment]
Yeah, that broke the 2.4 layout as well. Maybe they have it on their list of things to fix after the upgrade.

Thanks for the quick reply and for PRIMAP-hist 2.5!
